queue/queue.py

```diff
@@ -1,15 +1,29 @@
 class Queue:
-  def __init__(self):
-    self.size = 0
-    # what data structure should we
-    # use to store queue elements?
-    self.storage = 
-
-  def enqueue(self, item):
-    pass
-  
-  def dequeue(self):
-    pass
-
-  def len(self):
-    pass
+    def __init__(self):
+        self.size = 0
+        # what data structure should we
+        # use to store queue elements?
+        self.storage = []
+
+    def enqueue(self, item):
+        # add the item the to storage
+        self.storage.append(item)
+        # increase the size of the array by 1
+        self.size += 1
```

Comment:
Good job using the self.size prop provided to increment / decrement the size when adding & removing nodes. Linked list is a good choice in solving this prob, glad you're getting a hang of the data structures.